Source/WebCore/platform/graphics/gstreamer/MediaPlayerPrivateGStreamer.cpp

```diff
@@ -4600,8 +4612,16 @@ void MediaPlayerPrivateGStreamer::audioOutputDeviceChanged()
     if (!player)
         return;
 
+    auto* sink = audioSink();
+    if (!sink) {
+        GST_DEBUG_OBJECT(pipeline(), "No audio sink, skipping audio output device change");
+        return;
+    }
+
     auto deviceId = player->audioOutputDeviceId();
-    applyAudioSinkDevice(m_audioSink.get(), deviceId);
+    GST_DEBUG_OBJECT(pipeline(), "Switching to audio output device '%s'", deviceId.utf8().data());
+    [[maybe_unused]] bool succeeded = applyAudioSinkDevice(sink, deviceId);
+    GST_DEBUG_OBJECT(pipeline(), "%s to audio output device '%s' %s", succeeded ? "Changed to" : "Could not change", deviceId.utf8().data());
```

Comment:
```suggestion
    GST_DEBUG_OBJECT(pipeline(), "%s to audio output device '%s'", succeeded ? "Changed" : "Could not change", deviceId.utf8().data());
```
"to" is part of the fixed text, so "Changed" instead of "Changed to"